projects/kubernetes/kubernetes/1-19/patches/0012-EKS-PATCH-allow-override-of-kube-proxy-base-image

```diff
@@ -0,0 +1,30 @@
+diff --git a/build/common.sh b/build/common.sh
```

Comment:
Are there any sign off notes we want to include here, or are we just not including them anymore?

Reply:
Notes added.